src/classes/os-utils.ts

```diff
@@ -97,10 +97,11 @@ export default class OsUtils {
    * 
    * @returns Command Object which matches with the user OS and the searched command
    */
-  private static searchCmd (os: string, command: string) {
-    return OsCommands.getCmds()
+  private static async searchCmd (os: string, command: string) : Promise<string> {
```

Comment:
This method excepts that command is actaully found, but it's not enforced in any way. This could be achieved by changing the command from string to type that includes only valid types